specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json

```diff
@@ -5896,9 +5896,32 @@
             "modelAsString": true
           }
         },
+        "partnerTopicReadinessState": {
+          "description": "The readiness state of the corresponding partner topic.",
+          "enum": [
+            "NotActivatedByUserYet",
+            "ActivatedByUser",
+            "DeactivatedByUser",
+            "DeletedByUser"
```

Comment:
>ByUse [](start = 20, length = 5)

Any reason for suffixing with ByUser?

Reply:
The partner topic scenario involved two personas: publisher and customer or user. We need to differentiate between them in our resource lifecycle and we might add more reasons to that in the future.